src/core/cgroup.h

```diff
@@ -197,10 +207,12 @@ int unit_synthesize_cgroup_empty_event(Unit *u);
 int unit_get_memory_current(Unit *u, uint64_t *ret);
 int unit_get_tasks_current(Unit *u, uint64_t *ret);
 int unit_get_cpu_usage(Unit *u, nsec_t *ret);
+int unit_get_io_accounting(Unit *u, CGroupIOAccountingMetric metric, bool allow_cache, uint64_t *ret);
```

Comment:
I see `allow_cache` is `false` in the only caller.

Reply:
there are two callers: `property_get_io_counter()` in `dbus-unit.c` where it is passed as false. And `unit_log_resources()` in `unit.c` where it is true once and false for all subsequent iterations of the for() loop it is contained in.